### PR TITLE
Feat/auth google sign in

### PR DESCRIPTION
**Pull request by jakgrab**

### CODE REVIEW

File: feature-auth/src/main/java/com/example/auth/utils/Constants.kt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.example.auth.utils
+
+object Constants {
+}
```
Comment (bibiosm98) [inline feature-auth/src/main/java/com/example/auth/utils/Constants.kt]: why empty class?

File: feature-auth/src/main/res/layout/fragment_registration.xml [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        <!--            style="@style/RegisterButtonStyle"-->
+        <!--            android:layout_width="match_parent"-->
+        <!--            android:layout_height="60dp"-->
+        <!--            android:layout_marginVertical="20dp"-->
+        <!--            android:clickable="true"-->
+        <!--            android:text="@string/sign_in"-->
+        <!--            app:layout_constraintBottom_toBottomOf="parent"-->
+        <!--            app:layout_constraintEnd_toEndOf="parent"-->
+        <!--            app:layout_constraintStart_toStartOf="parent"-->
+        <!--             />-->
```
Comment (bibiosm98) [inline feature-auth/src/main/res/layout/fragment_registration.xml]: remove pls

**Review by bibiosm98 — APPROVED** [review]

File: feature-auth/src/main/java/com/example/auth/utils/Constants.kt (hunk 1 above)
Comment (jakgrab) [reply feature-auth/src/main/java/com/example/auth/utils/Constants.kt]: Moved to module-core, deleted

File: feature-auth/src/main/res/layout/fragment_registration.xml (hunk 2 above)
Comment (jakgrab) [reply feature-auth/src/main/res/layout/fragment_registration.xml]: ok

**Pull request merged**